pkg/param/param.go

```diff
@@ -293,14 +298,29 @@ func fetchRepositoryServer(ctx context.Context, cli kubernetes.Interface, crCli
 	if err != nil {
 		return nil, errors.Wrap(err, "Error Fetching Repository Server Service")
 	}
-	repositoryServerAddress := fmt.Sprintf("https://%s.%s.svc.cluster.local:%d", repositoryServerService.Name, repositoryServerService.Namespace, repositoryServerService.Spec.Ports[0].Port)
+	repositoryServerAddress := fmt.Sprintf("https://%s.%s.%s:%d", repositoryServerService.Name, repositoryServerService.Namespace, ClusterLocalDomain, repositoryServerService.Spec.Ports[0].Port)
+	contentCacheMB, metadataCacheMB := command.GetGeneralCacheSizeSettings()
+	if r.Spec.Repository.CacheSizeSettings.Content != "" {
+		contentCacheMB, err = strconv.Atoi(r.Spec.Repository.CacheSizeSettings.Content)
+		if err != nil {
+			return nil, errors.Wrap(err, "Error Parsing Content Cache Size")
+		}
+	}
+	if r.Spec.Repository.CacheSizeSettings.Metadata != "" {
+		metadataCacheMB, err = strconv.Atoi(r.Spec.Repository.CacheSizeSettings.Metadata)
+		if err != nil {
+			return nil, errors.Wrap(err, "Error Parsing Metadata Cache Size")
+		}
+	}
```

Comment:
Can we move it to a helper function, something like `getKopiaRepoCacheSize()`?

Reply:
Done !